Rakefile

```diff
@@ -4,8 +4,7 @@ rescue LoadError
   # github_changelog_generator is an optional group
 else
   GitHubChangelogGenerator::RakeTask.new :changelog do |config|
-    version = File.read('puppetboard/version.py').lines.last.scan(/(\d+\.\d+\.\d+)/).first.first
-    config.future_release = "v#{version}" if /^\d+\.\d+.\d+$/.match?(version)
+    config.future_release = `git rev-parse --abbrev-ref HEAD`.strip.split('-', 2).last
```

Comment:
This just parses the branch name? Not sure how reliable this is. for poetry, we could use `poetry version -s` (but then we need poetry installed :(

Reply:
this is how it works in my other repos when you stick to the brnach name pattern from RELEASE.md ... and peotry updates only the toml file, afaik. and the rake task ist still broken then because in the version.py is not version aynmore :shrug:

Reply:
```ruby
File.readlines('pyproject.toml').find{|l| l.match(/version/)}.split(' ').last.gsub('"', '')
```
also not very beautiful? :D

Reply:
If you like it this way. Don’t care. Just want a release. But also like my solution more. But won’t fight over it.